components/modal/modal.css

```diff
@@ -0,0 +1,56 @@
+:root {
+    --light-blue: #9AD0EC;
+}
+.modal-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: auto;
+    flex-direction: column;
+    margin: 0.5rem 0 1.5rem 0;
+}
+.modal-content {
+    padding: 1rem 1.5rem 0 1.5rem;
+    background-color: var(--white);
+    color: black;
+    visibility: hidden;
+    margin-top: -6rem;
+}
+.modal-header {
+    font-size: 1.4rem;
+    border-bottom: solid 2px darkgray;
```

Comment:
Color variables missing here only

Reply:
Thanks for pointing out that, changed it to color variable, please approve the changes